scripts/util.py

```diff
@@ -306,8 +350,9 @@ def build(project, folder, version_id, top_dir, path_dict, dependencies=None, fo
             shutil.rmtree(os.path.join(top_dir, "applet_resources", lang.upper()))
 
     region = dxpy.describe(project.get_id())['region']
-    asset_descs = dict(
+    asset_descs = [
```

Comment:
This is a list of tuples so with dict() removed it will fall over here: https://github.com/dnanexus/dxCompiler/blob/f7fe681d417a4ac74a32c10e11d735e375692fe7/scripts/build_release.py#L232 since `asset_ids` cannot be called on a list.

Reply:
fixed